Makefile

```diff
@@ -72,7 +72,7 @@ setup: clean
 	$Q go env
 
 ifndef HAS_GOLANGCI
-	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin v1.19.1
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin v1.30.0
```

Comment:
It looks like the `--deadline` flag has been replaced with `--timeout` in the newer version - https://github.com/golangci/golangci-lint/pull/793. Could you also update that as part of this PR?

Reply:
Done, updated also in `.golangci.yml` file